browser/brave_browser_process_impl.cc

```diff
@@ -264,6 +271,15 @@ BraveBrowserProcessImpl::https_upgrade_exceptions_service() {
   return https_upgrade_exceptions_service_.get();
 }
 
+localhost_permission_allowlist::LocalhostPermissionAllowlistService*
+BraveBrowserProcessImpl::localhost_permission_allowlist_service() {
+  if (!localhost_permission_allowlist_service_) {
```

Comment:
i gather you need to check the feature here

Reply:
I guess it's a question of whether the caller checks or not, but yeah, we should check here as well.